lib/active_model/serializer.rb

```diff
@@ -91,6 +92,8 @@ def self.get_serializer_for(klass, namespace = nil)
           serializer_class
         elsif klass.superclass
           get_serializer_for(klass.superclass)
+        else
+          nil
```

Comment:
this is really just making the current behavior explicit.  it means no serializer could be found. is it a good design? maybe not. has it been like this for a long time? yes.